tools/SetupFlow/DevHome.SetupFlow/ViewModels/SummaryViewModel.cs

```diff
@@ -106,7 +106,7 @@ public ObservableCollection<PackageViewModel> AppsDownloaded
     [RelayCommand]
     public void RemoveRestartGrid()
     {
-        _showRestartNeeded = Visibility.Collapsed;

```

Comment:
Bindings were never notified of this visibility update, was this working before the change?

Reply:
I'm inclined to assume this was a bug before.